rsts/community/index.rst

```diff
@@ -118,3 +118,332 @@ Flyte maintainers are available on Zoom for you to ask anything, weekly on Wedne
+**Example:** 
+
+.. code-block:: text
+
+    flytectl create project --id myflyteproject --name "My Flyte Project" --description "My very first project onboarding onto Flyte."
+    
+Starting a sentence with a component tool or component name
+___________________________________________________________
+* While starting a sentence with a tool or component name, prefer to use the right article before it.
+* Use a general descriptor for a component name.
+**Examples:**
+
+.. code-block:: text
+
+    - The myflyteapp directory comes with a sample workflow, which can be found under flyte/workflows/example.py.
+    - The Java/Scala SDK for Flyte.
+    - The @task decorator actually turns the decorated function into an instance of the PythonFunctionTask object.
+    - A workflow can contain a single functional node.
+    - ‘Flytectl allows you to communicate with FlyteAdmin using a YAML file or by passing every configuration value on the command-line’. (Donot use This allows you to …)
+
+Use code style for String and Integer field
+___________________________________________
+For string or integer values, use style (single backticks for rst and double backticks for markdown).
+
+**Examples:**
+
+.. code-block:: text
+
+    - Set the value of the field to True.
+
+Code Styling
+^^^^^^^^^^^^
+Use code style for
+
+* Filenames 
+* Directories
+* Paths
+* Commands
+* Inline code
+* Object field names
+* Namespaces
+* API objects
+* Anything that’s related to code!
+
+**Examples:**
+
+.. code-block:: text
+
+    - Commands: flytectl --help,flyte sandbox start,`flytectl exec`
+    - Filename: Open example.py in your favorite editor.
+    - Directory: The docs for this repo are present in /cookbook/docs folder
+    - Path: The myflyteapp directory comes with a sample workflow, which can be found under flyte/workflows/example.py.
+    - Inline code: Use flytectl get objectName, `create launchplan my_launchplan`
+    - Set value of `flytectl flag` to < >.
+    - `flytectl- apiserver …`
+    - The @task decorator actually turns the decorated function into an instance of the PythonFunctionTask object.
+* Include a title for the code example.
+* Use backtick (**`<inline code here>`**) for code style.
+* Enclose code samples with triple backticks. (```).
+* Use **..code-block::** a multi-line code block for fenced code blocks.
+
+**Example for code block:**
+
+.. code-block:: text
+
+    ```bash
+       Make html
+    ```
+
+    ..code-block:: python
+      :caption: this .py
+       :name: this-py
+       print 'Explicit is better than implicit.'
+
+* Use meaningful variable names that match the context.
+
+Python: Snake case
+Go: camelCase
+HTML: camelCase 
+
+* Remove trailing spaces in the code.
+* Begin with Upper case for inline comments (wherever applicable).
+* No period at the end of inline comments.
+* Code documentation should be precise.
+* Prefer active voice while explaining code.
+* Avoid filler words.
+
+Flytectl
+--------
```

Comment:
This isn't in the right structure. Flytectl, Common phrases ..., etc. should be under Code styling. Can you fix this?